### PR TITLE
Fix some issues

### PR DESCRIPTION
**Pull request by boeckMt**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/dlr-eoc/ukis-frontend-libraries/blob/master/CONTRIBUTING.md#-commit-message-guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)
- [x] The Project is [building and passes all tests](https://github.com/dlr-eoc/ukis-frontend-libraries/blob/master/DEVELOPMENT.md#further-you-can-test-and-build-locally) without errors

## PR Type
What kind of change does this PR introduce?

- [x] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [x] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A
- https://github.com/dlr-eoc/ukis-frontend-libraries/issues/112
- https://github.com/dlr-eoc/ukis-frontend-libraries/issues/119

## What is the new behavior?
- The two issues are fixed now.
- Refactored `create_..._layer` functions for less duplication when fixing #119.
- Some fixes to styling which is applied when creating a new application. They are also merged with the demo apps now.
- Changed the text about UKIS and the general structure of the main readme to better describe what this repo is for.
- Minor update to OpenLayers.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Other information
Is it part of one/more [packages](https://github.com/dlr-eoc/ukis-frontend-libraries/packages) and which?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,44 +5,55 @@
...
 - Use the frontend-libraries as npm modules (GitHub Packages) [see How to install Packages from GitHub](DEVELOPMENT.md)
+
 - or use the modules in typescript source via path mapping [see Developing libraries and frontend side by side](DEVELOPMENT.md)
 
 
+
+
+Libraries/packages and demo applications can be found in the [projects folder](projects).
+
+Here are e view examples:
```
Comment (MichaelLangbein) [inline README.md:47]: > e view

a few

File: README.md [hunk 2]
```diff
@@ -5,44 +5,55 @@
...
+- [A OpenLayers map component](projects/map-ol/README.md)
```
Comment (MichaelLangbein) [inline README.md:48]: > A

An

File: projects/demo-maps/src/app/app.component.ts [hunk 3]
```diff
@@ -5,6 +5,7 @@ import './components/icons/ukis';
 import { AlertService, IAlert } from './components/global-alert/alert.service';
 import { ProgressService, IProgress } from './components/global-progress/progress.service';
 import { Router, Routes } from '@angular/router';
+import { appVersion } from '../environments/version';
```
Comment (MichaelLangbein) [inline projects/demo-maps/src/app/app.component.ts:8]: Really useful! Thanks!

File: projects/map-ol/src/lib/map-ol.service.ts [hunk 4]
```diff
@@ -837,155 +892,66 @@ export class MapOlService {
...
-         * ol 6.x problem if popup (map.forEachLayerAtPixel) use className
-         * https://github.com/openlayers/openlayers/releases/tag/v6.0.0
-         */
-        layeroptions.className = l.id;
-      }
-
-      if (l.bbox) {
-        layeroptions.extent = transformExtent(l.bbox.slice(0, 4) as [number, number, number, number], WGS84, this.getProjection().getCode());
-      }
-
```
Comment (MichaelLangbein) [inline projects/map-ol/src/lib/map-ol.service.ts:962]: Really good to see all those `if` clauses removed. Thanks

**Review by MichaelLangbein — APPROVED** [review]
Looks a lot cleaner - thanks!

**Comment by boeckMt** [comment]
@MichaelLangbein thanks for your comments, I will update the PR to include these.

**Pull request merged**